tools/create-yorkie-app/README.md

```diff
@@ -0,0 +1,30 @@
+# create-yorkie-app
+
+## Usage
+
+You can scaffold yorkie-js-sdk examples by:
+
+```bash
+$ npx create-yorkie-app
+```
+
+The examples have own local dependencies. So you should install dependencies before running examples.
+
+```bash
+# In the directory of the example.
+$ npm install
+```
+
+Then you can run the examples.
+
+```bash
+# In the directory of the example.
+$ npm run dev
+```
```

Comment:
when I run the command I get an error `Error: Missing script: "dev"`. How can I use or test `create-yorkie-app`?

Reply:
Have you tried `npm run start` instead? I think we need to revise the document

Reply:
I wrote README.md for create-yorkie-app users not for developers. I will add documents for maintenance!